statsmodels/tsa/statespace/tests/test_dynamic_factor.py

```diff
@@ -473,8 +473,8 @@ def __init__(self):
     def test_mle(self):
         with warnings.catch_warnings(record=True) as w:
             mod = self.model
-            res = mod.fit(method='lbfgs', maxiter=10000, disp=-1)
-            res = mod.fit(res.params, method='nm', maxiter=10000, maxfev=10000, disp=False)
+            res1 = mod.fit(method='lbfgs', maxiter=10000, disp=-1)
+            res = mod.fit(res1.params, method='nm', maxiter=10000, maxfev=10000, disp=False)
```

Comment:
Yes sorry, I changed that when I was trying to understand where the error came from: from the computation of res or res1 ...